content/blog/2019/02/2019-02-05-jenkins-new-year-in-china.adoc

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title: "Jenkins new year in China"
+tags:
+- core
+- community
+author: linuxsuren
+---
+
+image:/images/2019-spring-festival.jpg[Spring Festival 2019, role=center]
+
+At the time of the Spring Festival. I want to make a summary of some activities in the last year. 
+You might already notice that more and more Chinese contributors emerge in the Jenkins community. 
+We have a GSoC champion who is link:https://github.com/cizezsy[Shenyu Zheng]. 
+He is a great example for other students. With the effort of three skilled engineers, 
+many Jenkins users could learn the edge technologies and useful use cases. 
+They co-orgnaized several Jenkins Meetups in a couple of cities in China.
+
+There are two workshops about Jenkins and Jenkins X in the DevOps International Summit. link:https://github.com/rawlingsj[James Rawlings] gave us a wonderful view of the Jenkins X. Many people start to know this project. The link:https://jenkins-x.io/zh/[Chinese website of jx] would be helpful to those people.
```

Comment:
I think `the Jenkins X` is better written as just `Jenkins X`.

Also, general recommendation: hard-wrapping after periods is a good thing to do to make the writings more diff friendly.